### DISCUSSION TITLE
Crop the SLCs error for StaMPS processing

### DISCUSSION THREAD

**Discussion by aripsnur** [alexisInSAR/EZ-InSAR#47] [Help]
Hi, I tried to run "Crop the SLCs" in StaMPS Processing. But error occurred in MATLAB command window as below:

ERROR 4: `lon.rdr.full' not recognized as a supported file format. 

ERROR 4: `lat.rdr.full' not recognized as a supported file format.
Traceback (most recent call last):
  File "/home/yongjekim/tools_insar/proc_insar/isce2/contrib/timeseries/prepStackToStaMPS/bin/crop_rdr.py", line 74, in <module>
    LonBand = LonData.GetRasterBand(1)
AttributeError: 'NoneType' object has no attribute 'GetRasterBand'

20181118
/data/Springfield/merged/SLC/20181118/20181118.slc.full 20181118.slc.full
/data/Springfield/merged/SLC/20181118/20181118.slc.full: Permission denied.
gdal2isce_xml.py -i 20181118.slc.full
This is the Open Source version of ISCE.
Some of the workflows depend on a separate licensed package.
To obtain the licensed package, please make a request for ISCE
through the website: https://download.jpl.nasa.gov/ops/request/index.cfm.
Alternatively, if you are a member, or can become a member of WinSAR
you may be able to obtain access to a version of the licensed sofware at
https://winsar.unavco.org/software/isce
Traceback (most recent call last):
  File "/home/arip/anaconda3/envs/InSARenv/lib/python3.8/site-packages/isce/applications/gdal2isce_xml.py", line 137, in <module>
    raise Exception('Input file is not found ....')
Exception: Input file is not found ....

Can you please help me to solve this problem?
Thanks.

**Comment by alexisInSAR** [comment]
Hi, 

Can you give us the list of files in the "merged" directory? 

Kind regards, 

Alexis

**Reply by geoenviron98** [reply]
Hi Alexis, 

I have a similar problem 
here is the file inside "merged"
![image](https://github.com/alexisInSAR/EZ-InSAR/assets/169086903/47d1765f-1fd3-46bc-848f-308372601524)

**Comment by aripsn** [comment]
Thank you for the response.

My solution is to repeat the step from zero.
I think I did not followed the instruction correctly or orderly, then messed up the folder.
It worked perfectly after repeating and following the instruction.

Sincerely,
Arip

**Comment by bspckishore** [comment]
Hi,

I have the same problem can you help?

**Reply by bspckishore** [reply]
I have successfully runed the ISCE processing, but when proceeding to Stamps, crop the SLC gives 
![image](https://github.com/alexisInSAR/EZ-InSAR/assets/138761855/913e3b27-abf9-4efd-83da-bd98cff98a36)

**Reply by alexisInSAR** [reply]
Hi, 

I think one of steps (during the InSAR processing) has failed. Please check that these files are available in the _merged_ directories, if not rerun the associated steps. 

Kind regards, 

Alexis

**Reply by bspckishore** [reply]
It works, thanks for your response :)

**Comment by geoenviron98** [comment]
Hi Alexis

I encountered this issue. Do you know how to solve this?

./scripttoeval_613b8c6d806c0ab9f6f7c445313c5514.sh: line 1: /home/s2495436/tools_insar/EZINSAR/EZINSAR_BIN//Suppfunctions/run_SLCcropStack_mod.csh: Permission denied

Regards,
Geo

**Reply by geoenviron98** [reply]
It can be solved using this command line:
chmod +x /home/s2495436/tools_insar/EZINSAR/EZINSAR_BIN//Suppfunctions/run_SLCcropStack_mod.csh

**Reply by Rofiatulumami** [reply]
Hi, i got the same problems. I change the permission, like you do. but, i still cant solved it. 
Is it possible if the problem in .sh file? cause, I've tried looking for the .sh file but it's still not there.

**Reply by geoenviron98** [reply]
Even though I got permission, the problem still existed. So I tried to run from the beginning again. And I didnt get any permission issues

**Comment by geoenviron98** [comment]
to handle this issue i thought you need re run the process carefully from
the beginning. I just did that and it worked

**Reply by keepfighting666** [reply]
hello, i use this py to crop ALOS-1 level 1.0, but showed the same error. Can you help me? The previous steps were processed in HPC.

**Reply by keepfighting666** [reply]
ok, i have fixed it.
